README.md

```diff
@@ -188,7 +188,7 @@ You can deploy this project using: Netlify & Github Pages
 
 ## 🔭 Project Presentation Outline <a name="loom"></a>
 
-- [Loom Video](loomlink)
+- [Loom Video](https://www.loom.com/share/087b1b664d0f41e9acbdba03e3fdf3d4)
```

Comment:
- Great Job on describing your project well in the video, however, you didn’t highlight a piece of the code that you find interesting or are proud of making it, please record another video with highlights code 👍